README.md

```
@@ -111,13 +122,13 @@ const uppercaseString = zagora()
 		uppercased: z.string(),
 	}))
 	.errors({
-  	network: z.object({
+  	NETWORK_ERROR: z.object({
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 3

(MD010, no-hard-tabs)